templates/interface/filter-panels.interface.hbs

```diff
@@ -65,75 +65,73 @@
         </div>
       </div>
 
-      <div id="default-comparison">
-        <div class="form-group clearfix" id="filter-value-type">
-          <div class="col-sm-4 control-label">
-            <label>Value type</label>
-          </div>
-          <div class="col-sm-8">
-            <label for="value-type-field-{{id}}" class="select-proxy-display">
-              <select name="value-type-field-{{id}}" id="value-type-field-{{id}}" data-field="valueType" class="hidden-select form-control">
-                <option value="enter-value">Enter a value</option>
-              </select>
-              <span class="icon fa fa-chevron-down"></span>
-            </label>
-          </div>
+      <div class="form-group clearfix" id="filter-value-type">
```

Comment:
The assumption in my previous review is that this `.hbs` template is repeated for each filter. That means this element ID is not unique to this filter. I mentioned all similar mistakes in this template needs to be fixed. Is this an exception for any good reason? If not, please fix them all in this file.

```suggestion
      <div class="form-group clearfix">
```

Reply:
@tonytlwu 
I added this one identifier to add a "hidden" class to the block, which is  used in other logic options, but during the choice "between logic" I added 2 of the same blocks with changed labels, in my opinion, this is clearer and clearer for further work with logic

Reply:
I'm not sure why there's any further discussion required around this.

[The file](https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/473/files#diff-6580d9e7b0e71f2c9a22ce6a2103b2bc6aced2c8c25fc51a3271104665c7fd41) is repeated for each filter, therefore there the template should not contain any `id` attribute that isn't unique to each repetition because it will create 2 elements of the same ID -- we can't have this.

Reply:
@tonytlwu 
I added an ID for the required tags.

Reply:
There are still elements in the file that has an `id` that gets repeated.

https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/4e2652dd518eab3632968dc1a056717ebd401bbe/templates/interface/filter-panels.interface.hbs#L87

Reply:
@tonytlwu, sorry missed this moment when resolved conflicts